lib/runtime/wasmer/instance.go

```diff
@@ -94,6 +97,22 @@ func NewInstance(code []byte, cfg *Config) (*Instance, error) {
 		return nil, errors.New("code is empty")
 	}
 
+	// Substrate Wasm compression
+	// https://github.com/paritytech/substrate/blob/master/primitives/maybe-compressed-blob/src/lib.rs
+	const compressionFlag = []byte{82, 188, 83, 118, 70, 219, 142, 5}
```

Comment:
I don't think you can have slices as constants (very unfortunately)

```suggestion
	compressionFlag := []byte{82, 188, 83, 118, 70, 219, 142, 5}
```